internal/server/router/router.go

```diff
@@ -48,11 +54,26 @@ func NewRouter(config Config) (*Router, error) {
 	}, nil
 }
 
-func (a *Router) IngestEvents(w http.ResponseWriter, r *http.Request) {
-	a.config.IngestHandler.ServeHTTP(w, r)
+// FIXME: openapi generator doesn't generate an input param
```

Comment:
@tothandras openapi generated doesn't add an input param here like for meter. Any idea why?